### PR TITLE
fix #133247 photosonic.writesonic.com

### LINKED ISSUE #133247: photosonic.writesonic.com
### Issue URL (Ads)
 [https://photosonic.writesonic.com/?via=artturi](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fphotosonic.writesonic.com%2F%3Fvia%3Dartturi)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.9
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180)
DNS filtering: | server: `https://dns.adguard.com/dns-query`<br>filters: `https://filters.adtidy.org/windows/filters/15.txt, User rules`
Filters: | <b>Ad Blocking:</b><br/>EasyList, <br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>EasyPrivacy, <br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>Fanboy's Social Blocking List, <br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>Fanboy's Annoyances, <br/>AdGuard Annoyances<br/><br/><b>Other:</b><br/>AdGuard Experimental<br/><br/><b>Language-specific:</b><br/>List-KR, <br/>AdGuard French, <br/>AdGuard Russian, <br/>AdGuard Chinese, <br/>AdGuard German, <br/>AdGuard Japanese, <br/>AdGuard Spanish/Portuguese
Userscripts: | https://kill.org/beta/adguard-extra/1.0/adguard-extra.user.js,<br>https://userscripts.adtidy.org/beta/popup-blocker/2.5/popupblocker.user.js

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [X] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/133247

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: AnnoyancesFilter/Popups/sections/antiadblock.txt [hunk 1]
```diff
@@ -3189,6 +3190,7 @@ dniwolne.eu#$#.adsbygoogle { height: 1px!important; }
 ! SECTION: Anti-adblock - The rules for content blockers only
 !#if (adguard_ext_safari || adguard_app_ios || adguard_ext_android_cb)
 !#safari_cb_affinity(general)
+@@||www3.doubleclick.net/|$domain=photosonic.writesonic.com
```
Comment (Alex-302) [inline AnnoyancesFilter/Popups/sections/antiadblock.txt:3193]: Better with content type
```suggestion
@@||www3.doubleclick.net/|$xmlhttprequest,domain=photosonic.writesonic.com
```

File: AnnoyancesFilter/Popups/sections/antiadblock.txt [hunk 2]
```diff
@@ -631,6 +631,7 @@ reddit.com#@#.ADBAR
 ! SECTION: Anti-adblock - Regular rules
 ! Do not forget about '!#safari_cb_affinity' section for exclusions
 !
+||www3.doubleclick.net/|$redirect=nooptext,domain=photosonic.writesonic.com
```
Comment (Alex-302) [inline AnnoyancesFilter/Popups/sections/antiadblock.txt:634]: Better with content type
```suggestion
||www3.doubleclick.net/|$xmlhttprequest,redirect=nooptext,domain=photosonic.writesonic.com
```
also can be used this scriptlet(also works in Safari)
`photosonic.writesonic.com#%#//scriptlet('prevent-fetch', 'www3.doubleclick.net')`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/Popups/sections/antiadblock.txt (hunk 2 above)
Comment (piquark6046) [reply AnnoyancesFilter/Popups/sections/antiadblock.txt]: Done.

File: AnnoyancesFilter/Popups/sections/antiadblock.txt (hunk 1 above)
Comment (piquark6046) [reply AnnoyancesFilter/Popups/sections/antiadblock.txt]: Done.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**